core/tests/memory/any_value.cpp

```diff
@@ -35,6 +35,14 @@ TEST_CASE("memory::AnyValue")
             CHECK(any.type().is<int>());
             CHECK(*static_cast<int*>(any.get()) == value);
         }
+
+        SUBCASE("move assignment")
+        {
+            int value = 1337;
+            AnyValue any = AnyValue::moveConstruct(reflect<int>(), &value);
+            CHECK(any.type().is<int>());
+            CHECK(*static_cast<int*>(any.get()) == value);
```

Comment:
I would change this to
```suggestion
            int value = 1337;
            AnyValue any{};
            REQUIRE_FALSE(any.valid());
            any = AnyValue::moveConstruct(reflect<int>(), &value);
            REQUIRE(any.valid());
            CHECK(any.type().is<int>());
            CHECK(*static_cast<int*>(any.get()) == value);
```